docs/zh_cn/multi_modal/llava.md

```diff
@@ -2,14 +2,13 @@
 
 LMDeploy 支持以下 LLaVA 系列模型，具体如下表所示：
 
-|                 模型                 | 大小 |   支持的推理引擎   |
-| :----------------------------------: | :--: | :----------------: |
-| llava-hf/Llava-interleave-qwen-7b-hf |  7B  | TurboMind, PyTorch |
```

Comment:
why remove llava-hf/Llava-interleave-qwen-7b-hf, llava-hf/llava-v1.6-mistral-7b-hf and llava-hf/llava-v1.6-vicuna-7b-hf for pytorch engine